src/main/java/net/coderbot/iris/pipeline/HandRenderer.java

```diff
@@ -121,10 +127,18 @@ public void renderTranslucent(RenderBuffers renderBuffers, PoseStack poseStack,
 
 		gameRenderer.resetProjectionMatrix(CapturedRenderingState.INSTANCE.getGbufferProjection());
 
+		bufferSource.endBatch();
+
+		bufferSource.resetDrawCalls();
```

Comment:
I don't think this does anything particularly important, but it would be good to have it in the other method too to match up.

Reply:
We don't want to reset the draw call counter before we've finished everything.

Reply:
But translucent hand rendering is never guaranteed to be executed.

Reply:
That's true, is it good to reset the draw call count when finalizing level rendering?